libtextsecure/helpers.js

```diff
@@ -90,6 +90,11 @@ window.textsecure.utils = function() {
         return number.split(".");
     };
 
+    self.isNumberSane = function(number) {
```

Comment:
Always better to whitelist instead of blacklist, especially when we have a well-defined whitelist already :)